### DISCUSSION TITLE
Trying to Configure Hyper HDR with RPI 3B+

### DISCUSSION THREAD

**Discussion by DSELCTD** [awawa-dev/HyperHDR#817] [Q&A] [answered]
Hello,

Trying a swing at this HyperHDR project again, did some research and found that I needed to get a 3.3V to 5V Level Shifter to make this project viable and so the RPI would read the LEDs. I am using WS2812B LEDS. I have everything up to date. HyperHDR v20.0.0.0, I have the logic shifter set up for data wire to GPIO 18 and the respective power and ground for the Pi. For whatever reason it is still not reading my LED device. These are in my logs. I am not sure how to run HyperHDR as a root.

2024-04-04T14:30:48.094Z [LEDDEVICE0_WS281X] Device 'ws281x' is disabled due to an error: 'Failed to open. Error message: mmap() failed. YOUR NEED TO RUN HYPERHDR AS A ROOT USER FOR MMAP TO WORK. SUCH CONFIGURATION IS NOT OFFICIALLY SUPPORTED.'

**Comment by awawa-dev** [comment, marked as answer]
Hi
There is a link to instructions on switching the user to root on the wiki https://github.com/awawa-dev/HyperHDR/wiki/FAQ

**Reply by DSELCTD** [reply]
Done, now I cannot access the website via IP.

**Reply by DSELCTD** [reply]
Nevermind! All good now, thanks!

**Comment by empakoso** [comment]
My RPi 3B working with RP2040 handling all the level shifting and bit banging!  Keeps up really nicely and all fits behind my TV
![IMG_7236](https://github.com/awawa-dev/HyperHDR/assets/26410171/0219c3f9-e463-4a4d-9a44-3245bcbab2b0)
